src/App.js

```diff
@@ -0,0 +1,315 @@
+//to get this started, change to directory and run "npm run start"
+
+import React from 'react';
+import {useState} from 'react';
+
+import SeedRandom from 'seedrandom';
+
+import SentenceData from './HoroscopeSentences.json' with { type: 'json' };
+import StarsignData from './Starsigns.json' with { type: 'json' };
+
+import Genie from './Genie.js';
+
+import Button from '@mui/material/Button';
+import DateInput from './DateInput.js';
+import GetCurrentDate from './GetCurrentDate.js';
+
+import './App.css';
+
+function App() {
+  const minsentences = 4;
+  const maxsentences = 8;
+  
+  const [spindir, setSpindir] = useState('App-logo');
+
+  const makeGenieSpin = () => {
+    var tempspindir = Math.random();
+    var mynum = tempspindir
+    if (tempspindir < 0.5) {
+      tempspindir = 'Cw';
+    } else {
+      tempspindir = 'Ccw';
+    }
+    setSpindir(mynum+" "+tempspindir);
+    
+  }
+
+  function returnCurrentDate() {
+    const curdate = new Date().toISOString().split("T")[0];
+    return(
+      curdate
+    );
+  }
+
+  const [starsign, setStarsignText] = useState();
+  const onBirthdayDateChange = (event) => {
+    //set the starsign.
+
+    const base = event.target.value
+    let day = base.split("-")[2];
+    const month = base.split("-")[1];
+    const year = base.split("-")[0];
```

Comment:
This could be better written as 
```suggestion
    const [year, month, day] = base.split("-");
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring#description

Reply:
I do agree it is cleaner though.

Reply:
Also, should be `let` not `const`. `day` is being set below so it can't be a `const`.